<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtocolsConfig</class>
 <widget class="QWidget" name="ProtocolsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>241</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pop Up Notices</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Application notifications</string>
     </property>
     <property name="buddy">
      <cstring>showNotifications</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="showNotifications">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>File transfers and other jobs</string>
     </property>
     <property name="buddy">
      <cstring>showJobs</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="showJobs">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Icons</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Application status</string>
     </property>
     <property name="buddy">
      <cstring>showApplicationStatus</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="showApplicationStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Communications</string>
     </property>
     <property name="buddy">
      <cstring>showCommunications</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="showCommunications">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>System services</string>
     </property>
     <property name="buddy">
      <cstring>showSystemServices</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="showSystemServices">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Hardware control</string>
     </property>
     <property name="buddy">
      <cstring>showHardware</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="showHardware">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
